tests/phpunit/tests/multisite/ms-functions/update_posts_count.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+if ( is_multisite() ) :
+	/**
+	 * Test update_posts_count() get called via filters of WP_Site in multisite.
+	 *
+	 * @group ms-site
+	 * @group multisite
+	 *
+	 * @covers ::_update_posts_count_on_delete
+	 */
+	class Tests_update_posts_count_on_delete extends WP_UnitTestCase {
+
+		/**
+		 * This test that the posts count is updated correctly when a posts are added and deleted.
+		 * @ticket 53443
+		 */
+		public function test_update_posts_count_on_delete() {
+
+			$blog_id = self::factory()->blog->create();
+			switch_to_blog( $blog_id );
+
+			$current_post_count = (int) get_option( 'post_count' );
+
+			$post_id = self::factory()->post->create(
+				array(
+					'post_type'   => 'post',
+					'post_author' => '1',
+					'post_date'   => '2012-10-23 19:34:42',
```

Comment:
Adding `'post_status' => 'publish'` should fix the failing tests

Reply:
good idea